src/config.js

```diff
@@ -1,9 +1,9 @@
-let address = 'https://skybunk-development.herokuapp.com';
+let address = 'https://skybunk-auth-dev.herokuapp.com';
```

Comment:
skybunk-auth-dev is the right address? Should we have an skybunk-auth-prod?

Reply:
The line below this uses the `AUTH_ADDRESS` environment variable if it exists. On the production server, I have that environment variable set to prod